lib/assessment.js

```diff
@@ -20,9 +38,17 @@ var sum = function sum() {
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
+  //create a new empty array
+  //loop through length and push default Value to the new empty arrays
+    var Array = [];
+    for(var i = 0; i < length; i++){
 
+     Array[i] = defaultValue;
+    return Array;
```

Comment:
The `return` statement will need to be outside of the `for` loop in order for this to work correctly.